MapboxVision/src/main/java/com/mapbox/vision/video/videosource/file/FileVideoDecoder.kt

```diff
@@ -25,6 +27,8 @@ class FileVideoDecoder(
     private var videoProgressTimestamp = 0L
     private var playStartTimestamp = 0L
 
+    private val paused : AtomicBoolean = AtomicBoolean(false)
```

Comment:
is it provided by API? I think iOS doesn't support this feature

Reply:
Nope, added for internal testing, will remove.